ui/js/controllers/perf/alerts.js

```diff
@@ -108,30 +135,57 @@ perf.controller('AlertsCtrl', [
                 alertSummary.allSelected = false;
             }
         };
-        $scope.reassignAlerts = function(alertSummary) {
-            var modalInstance = $modal.open({
-                templateUrl: 'partials/perf/reassignalertsctrl.html',
-                controller: 'ReassignAlertsCtrl',
+
+        $scope.fileBug = function(alertSummary) {
+            window.open("https://edmorley.github.io/fileit/");
```

Comment:
I would like a todo item to enhance this- there is a lot we can do here to reduce the pain of filing bugs.